pkg/driver/node/mounter/pod_mounter.go

```diff
@@ -138,14 +138,6 @@ func (pm *PodMounter) Mount(ctx context.Context, bucketName string, target strin
 	podMountSockPath := mppod.PathOnHost(podPath, mppod.KnownPathMountSock)
 	podMountErrorPath := mppod.PathOnHost(podPath, mppod.KnownPathMountError)
 
-	klog.V(4).Infof("Waiting for Mountpoint Pod %s to be ready to accept connections on %s", pod.Name, podMountSockPath)
-
-	// Wait until Mountpoint Pod is ready to accept connections
-	err = util.WaitForUnixSocket(mountpointPodReadinessTimeout, mountpointPodReadinessCheckInterval, podMountSockPath)
```

Comment:
Why don't we need this anymore?

Reply:
> With this PR, mountoptions.Send automatically retries on both connect: connection refused and file does not exist and removes util.WaitForUnixSocket as that's already handled with mountoptions.Send.

Basically the retry on `errors.Is(err, syscall.ENOENT)` achieves the same thing as `util.WaitForUnixSocket` and it's no longer needed.